src/aws/resourcemanager/resourcemanager.py

```diff
@@ -1,16 +1,104 @@
 """
 Module for the Resource Manager.
 """
+import sys
+sys.path.append('./src')
+
 from aws.utils.monitor import Listener
+import uuid
+import boto3
+from botocore.exceptions import ClientError, DataNotFoundError
+import os
 
 
 class ResourceManagerCore:
 
     def __init__(self):
-        pass
+        self.S3 = boto3.client('s3')
+        self.S3_resource = boto3.resource('s3')
+        self.S3_Session = boto3.session.Session()
 
     def run(self):
-        raise NotImplementedError()
+        try:
+            # Check the bucket list
+            bucket_list = self.S3.list_buckets()["Buckets"]
+            # If there are no buckets, create a new one
+            if not bucket_list:
+                name,response = self.create_bucket()
+                bucket_list = self.S3.list_buckets()["Buckets"]
+            # Use the first bucket in the list
+            bucket_name = bucket_list[0]["Name"]
+            """
+            Lines below is an example of what you can do with this code.
+            """
+            key = 'text'
+            if os.path.exists('src/aws/resourcemanager/upload.txt'):
```

Comment:
This hardcoded file should be changed later on. For this PR it is fine

Reply:
Yeah these are just examples of what can be done, and as you probably don't have such a file, the code would fail otherwise. In upload_file itself, the check is done with the file_path given to the function, so that works on its own